frontend/lib/src/components/elements/Metric/Metric.tsx

```diff
@@ -42,85 +41,74 @@ export interface MetricProps {
 export default function Metric({
   element,
 }: Readonly<MetricProps>): ReactElement {
-  const { colors }: EmotionTheme = useTheme()
-  const { MetricColor, MetricDirection } = MetricProto
+  const { MetricDirection } = MetricProto
+  const {
+    body,
+    label,
+    delta,
+    direction,
+    color,
+    labelVisibility,
+    help,
+    border,
+  } = element
 
-  let direction: any = null
-  let color = ""
+  let metricDirection: any = null
```

Comment:
Is it possible to have better typing here than `any`? It looks like it can be one of `ArrowDownward`, `ArrowUpward` or `null`